edge-util/src/Microsoft.Azure.Devices.Edge.Storage.RocksDb/ColumnFamilyDbStore.cs

```diff
@@ -54,13 +63,16 @@ public Task Put(byte[] key, byte[] value, CancellationToken cancellationToken)
             Preconditions.CheckNotNull(key, nameof(key));
             Preconditions.CheckNotNull(value, nameof(value));
 
+            this.count += 1;
             Action operation = () => this.db.Put(key, value, this.Handle);
             return operation.ExecuteUntilCancelled(cancellationToken);
         }
 
         public Task Remove(byte[] key, CancellationToken cancellationToken)
         {
             Preconditions.CheckNotNull(key, nameof(key));
+
+            this.count -= 1;
```

Comment:
1. Should this be capped at 0 as min or underflow? 
2. Maybe we can consider `unsigned long` type for `this.count`?

Reply:
I think that is a great idea :)